examples/ContinuousMotion.py

```diff
@@ -0,0 +1,13 @@
+# output: ContinuousMotion
+# name: ContinuousMotion
+# visible: False
+# type: video
+
```

Comment:
Does not have an import statement.

Reply:
It's added in the rendered page it there is none here.

Reply:
Then the other examples should not have import statements.

Reply:
hmm, will add them then

Reply:
In line with the docs, there should also only be one blank line between the import statement and the class

Reply:
> In line with the docs, there should also only be one blank line between the import statement and the class

^

Reply:
It's actually black which formatted the file like that.

Reply:
Is black insistent about the two lines of whitespace?

Reply:
Yeah